src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj

```diff
@@ -27,4 +27,16 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
+
```

Comment:
Please remove `TargetFrameworks` from this file
Add ` <TreatWarningsAsErrors>true</TreatWarningsAsErrors>` and fix errors if any are reported

Reply:
Changed